tests/fixtures/1_single_zone_tiny_house.json

```diff
@@ -0,0 +1,961 @@
+                    "roughness": "Smooth",
+                    "thickness": 0.0015,
+                    "conductivity": 45.0,
+                    "density": 7690.0,
+                    "specific_heat": 410.0,
+                    "thermal_absorptance": 0.9,
+                    "solar_absorptance": 0.5,
+                    "visible_absorptance": 0.5
+                },
+                {
+                    "type": "EnergyMaterial",
+                    "name": "Thick Stone",
+                    "roughness": "Rough",
+                    "thickness": 0.3,
+                    "conductivity": 2.31,
+                    "density": 2322.0,
+                    "specific_heat": 832.0,
+                    "thermal_absorptance": 0.95,
+                    "solar_absorptance": 0.75,
+                    "visible_absorptance": 0.8
+                }
+            ]
+        }
+    },
+    "rooms": [
+        {
+            "type": "RoomAbridged",
+            "name": "TinyHouseZone",
+            "name_original": "Tiny House Zone",
+            "properties": {
+                "type": "RoomProperties",
+                "energy": {
+                    "type": "RoomEnergyPropertiesAbridged",
+                    "program_type": null,
+                    "construction_set": null,
+                    "people": null,
+                    "lighting": null,
+                    "electric_equipment": null,
+                    "gas_equipment": null,
+                    "infiltration": null,
+                    "ventilation": null
+                }
+            },
+            "faces": [
+                "TinyHouseZone_Bottom",
+                "TinyHouseZone_Front",
+                "TinyHouseZone_Right",
+                "TinyHouseZone_Back",
+                "TinyHouseZone_Left",
+                "TinyHouseZone_Top"
+            ],
+            "indoor_shades": [
+                "TinyHouseZone_Back_InShd0"
+            ],
+            "outdoor_shades": [
+                "TinyHouseZone_Back_OutShd0"
+            ]
+        }
+    ],
+    "faces": [
+        {
+            "type": "FaceAbridged",
+            "name": "TinyHouseZone_Bottom",
+            "name_original": "Tiny House Zone_Bottom",
+            "geometry": {
+                "type": "Face3D",
+                "boundary": [
+                    [
+                        0.0,
+                        0.0,
+                        0.0
+                    ],
+                    [
+                        0.0,
+                        10.0,
+                        0.0
+                    ],
+                    [
+                        5.0,
+                        10.0,
+                        0.0
+                    ],
+                    [
+                        5.0,
+                        0.0,
+                        0.0
+                    ]
+                ]
+            },
+            "properties": {
+                "type": "FaceProperties",
```

Comment:
Why are some properties (e.g. `energy::construction` in here but others are not `boundary_condition`)?

Reply:
`boundary_condition` is something that we realized needs to be shared between Radiance and Energy models since, particularly when it comes to 3-phase daylight, you need to know when an Aperture is next to another room or directly sees the sky.  So the `face_type` and `boundary_condition` exist on the core Face object and only the properties specifically related to energy go here under this section.  Right now, this is only the construction.

Reply:
In other words, to be able to make the distinction between indoor and outdoor faces and windows we need to have access to boundary condition in higher level.